server/graphql/types/src/types/program/patient.rs

```diff
@@ -217,6 +217,25 @@ impl PatientNode {
         })
     }
 
+    pub async fn next_of_kin(&self, ctx: &Context<'_>) -> Result<Option<PatientNode>> {
+        let loader = ctx.get_loader::<DataLoader<PatientLoader>>();
+
+        if let Some(next_of_kin_id) = &self.patient.next_of_kin_id {
+            let result = loader
```

Comment:
probably copy pasta things but nit: 

```suggestion
    pub async fn next_of_kin(&self, ctx: &Context<'_>) -> Result<Option<PatientNode>> {
        if let Some(next_of_kin_id) = &self.patient.next_of_kin_id {
            let loader = ctx.get_loader::<DataLoader<PatientLoader>>();
            let result = loader
```

Don't need to initialise the loader for the other branch?

Reply:
[3d8dde3](https://github.com/msupply-foundation/open-msupply/pull/6375/commits/3d8dde39240d8d59ed2addf7a92f4ac07ef589e4)